proco/locations/serializers.py

```diff
@@ -51,10 +51,10 @@ def get_integration_status(self, instance):
         return instance.latest_status[0].integration_status if instance.latest_status else None
 
     def get_date_of_join(self, instance):
-        return getattr(instance, 'date_of_join', None)
+        return instance.first_status[0].created if instance.first_status else None
```

Comment:
let's save date of join directly to the country so we won't need additional db query just to get one field. this information will be filled at once on first csv upload and then will never be changed